src/current/v23.2/create-virtual-cluster.md

```diff
@@ -0,0 +1,110 @@
+---
+title: CREATE VIRTUAL CLUSTER
+summary: The CREATE VIRTUAL CLUSTER statement creates a new target virtual cluster for physical replication.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+{% include_cached new-in.html version="v23.2" %} The `CREATE VIRTUAL CLUSTER` statement starts a [physical cluster replication job]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}). `CREATE VIRTUAL CLUSTER` only supports starting a replication stream as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
+
+{% include {{ page.version.version }}/physical-replication/phys-rep-sql-pages.md %}
+
+## Required privileges
+
+`CREATE VIRTUAL CLUSTER` requires one of the following privileges:
+
+- The `admin` role.
+- The `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows the user to run all the related `VIRTUAL CLUSTER` SQL statements for physical cluster replication.
+- The `REPLICATION` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows a user to connect to stream data out of a virtual cluster for physical cluster replication.
```

Comment:
This permission does not allow you to run CREATE VIRTUAL CLUSTER (or at least, it shouldn't).

Reply:
Ah! I misunderstood this. We have had a convo in slack about this. It seems like the only user-facing use of `REPLICATION` is to grant to the primary cluster user so there is a user profile to connect to from the standby cluster to start the stream. So, I have removed the `replication` priv from this page. Thanks for pointing out!